pkg/cmd/roachtest/roachtestutil/mixedversion/steps.go

```diff
@@ -969,3 +971,53 @@ func (s networkPartitionRecoveryStep) Run(
 func (s networkPartitionRecoveryStep) ConcurrencyDisabled() bool {
 	return false
 }
+
+type alterReplicationFactorStep struct {
+	replicationFactor int
+	targetNode        option.NodeListOption
+}
+
+func (s alterReplicationFactorStep) Background() shouldStop { return nil }
+
+func (s alterReplicationFactorStep) Description() string {
+	return fmt.Sprintf("alter replication factor to %d", s.replicationFactor)
+}
+
+func (s alterReplicationFactorStep) Run(
+	ctx context.Context, l *logger.Logger, rng *rand.Rand, h *Helper,
+) error {
+	stmt := fmt.Sprintf("ALTER RANGE default CONFIGURE ZONE USING num_replicas = %d", s.replicationFactor)
+	if err := h.System.Exec(
+		rng,
+		stmt,
+	); err != nil {
+		return errors.Wrap(err, "failed to change replication factor on system")
+	}
+	if h.IsMultitenant() {
```

Comment:
Do we need to change the RF for separate process tenants? I don't think so but I could be wrong?

Reply:
Forgot to remove this from when we were debugging the long replication time, should not need to change RF for separate process tenants - removed